### PR TITLE
Fix #460: Incorrect highlighting for multiline imports

### LINKED ISSUE #460: Incorrect highlighting of multiline import statements
Currently multi-line import statements aren't highlighted correctly;

```js
import foo from 'bar';

import foo
from 'bar'

import foo
  from 'bar'
```

screen shot 2016-11-14 at 17 11 50

This is on the master branch.

### PR DESCRIPTION
**Pull request by Alhadis**
Fixes the issues brought up by @bgriffith in #460, as well as another I noticed while filling out specs:

```js
import { default as alias };

import
{
default
as
alias
};
```

### CODE REVIEW

File: grammars/javascript.cson [hunk 1]
```diff
@@ -133,7 +122,7 @@
         'name': 'keyword.control.js'
       }
       {
-        'match': '\\b([a-zA-Z_$][\\w$]*)\\b(?=.*\\bfrom\\b)'
+        'match': '\\b([a-zA-Z_$][\\w$]*)\\b(?=.*(?:$|\\bfrom\\b))'
```
Comment (MaximSokolov) [inline grammars/javascript.cson]: `(?=.*(?:$|\\bfrom\\b))` is not needed here

File: spec/javascript-spec.coffee [hunk 2]
```diff
@@ -741,12 +741,43 @@ describe "JavaScript grammar", ->
...
+
     it "tokenizes default import", ->
       {tokens} = grammar.tokenizeLine('import defaultMember from "module-name";')
       expect(tokens[0]).toEqual value: 'import', scopes: ['source.js', 'meta.import.js', 'keyword.control.js']
       expect(tokens[2]).toEqual value: 'defaultMember', scopes: ['source.js', 'meta.import.js', 'variable.other.module.js']
       expect(tokens[4]).toEqual value: 'from', scopes: ['source.js', 'meta.import.js', 'keyword.control.js']
 
+      lines = grammar.tokenizeLines """
+        import
+          \\x20{2}
```
Comment (MaximSokolov) [inline spec/javascript-spec.coffee]: It should be `\x20\x20`. I don't think testing for whitespaces is necessary, as it's already separated by lines

File: spec/javascript-spec.coffee [hunk 3]
```diff
@@ -757,6 +788,31 @@ describe "JavaScript grammar", ->
       expect(tokens[10]).toEqual value: '}', scopes: ['source.js', 'meta.import.js', 'punctuation.definition.modules.end.js']
       expect(tokens[12]).toEqual value: 'from', scopes: ['source.js', 'meta.import.js', 'keyword.control.js']
 
+      lines = grammar.tokenizeLines """
+        import
+        \\t
```
Comment (MaximSokolov) [inline spec/javascript-spec.coffee]: Same here

File: spec/javascript-spec.coffee (hunk 2 above)
Comment (Alhadis) [inline spec/javascript-spec.coffee]: The whitespace is only to stop the whitespace package stripping empty lines or something, which is what @50Wliu pointed out in [another pull request](https://github.com/atom/language-css/pull/99#issuecomment-260448878).

The blank lines are also to ensure multiline-matching isn't restricted to the following line only.

File: spec/javascript-spec.coffee (hunk 3 above)
Comment (MaximSokolov) [inline spec/javascript-spec.coffee]: Also `\\t` -> `\t`

Comment (Alhadis) [inline spec/javascript-spec.coffee]: I clearly can't edit a language grammar while compiling a V8 build, it seems. )))))))))))

Pushed an amended commit.

File: spec/javascript-spec.coffee [hunk 4]
```diff
@@ -741,12 +741,43 @@ describe "JavaScript grammar", ->
...
+
     it "tokenizes default import", ->
       {tokens} = grammar.tokenizeLine('import defaultMember from "module-name";')
       expect(tokens[0]).toEqual value: 'import', scopes: ['source.js', 'meta.import.js', 'keyword.control.js']
       expect(tokens[2]).toEqual value: 'defaultMember', scopes: ['source.js', 'meta.import.js', 'variable.other.module.js']
       expect(tokens[4]).toEqual value: 'from', scopes: ['source.js', 'meta.import.js', 'keyword.control.js']
 
+      lines = grammar.tokenizeLines """
+        import
+          \\x20
```
Comment (MaximSokolov) [inline spec/javascript-spec.coffee]: It's should be `\x20` not `\\x20`. You can add test for `lines[1][0]` to make sure

File: spec/javascript-spec.coffee [hunk 5]
```diff
@@ -741,12 +741,43 @@ describe "JavaScript grammar", ->
       expect(tokens[4]).toEqual value: '"', scopes: ['source.js', 'meta.import.js', 'string.quoted.double.js', 'punctuation.definition.string.end.js']
       expect(tokens[5]).toEqual value: ';', scopes: ['source.js', 'punctuation.terminator.statement.js']
 
+      lines = grammar.tokenizeLines """
+        import
+          \\x20{2}
```
Comment (MaximSokolov) [inline spec/javascript-spec.coffee]: Also here

File: spec/javascript-spec.coffee [hunk 6]
```diff
@@ -776,6 +832,55 @@ describe "JavaScript grammar", ->
...
+      lines = grammar.tokenizeLines """
+        import
+        {
+        \\t
+          member
+        \\t
+        }
+        \\t
+          from
+        \\t
```
Comment (MaximSokolov) [inline spec/javascript-spec.coffee]: And here

**Pull request merged**